src/utils/list.jsx

```diff
@@ -1,40 +1,52 @@
 import React, {useReducer} from 'react';
 import '../css/list.css';
 import {Trash2} from 'react-feather';
-
+import {Title} from '@statisticsnorway/ssb-component-library';
```

Comment:
Unused import. It's probably not yours, it would be nice if you remove it. Thanks :D